packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -52,6 +52,26 @@ $detailsList-item-focus-meta-text-color: $ms-color-neutralDark;
   }
 }
 
+/*
+ * When the DetailsList hover mode is "above" and the list is in hover state,
+ * style all rows with "hover" colors by default and
+ * use default colors in rows below the hovered row.
+ */
+:global(.ms-DetailsList--HoverAbove:hover) {
```

Comment:
why are we adding global css??? This isn't versionable.

I see it above and below this... that was a mistake and should not be copied.